src/lib/datepicker/calendar.ts

```diff
@@ -185,8 +213,11 @@ export class MatCalendar<D> implements AfterContentInit, OnDestroy, OnChanges {
   }
 
   ngAfterContentInit() {
+
```

Comment:
nit: no blank line

Reply:
done in a88fecbec2c3292017613bf4e75c38ff7c632042